app/routes/routes.py

```diff
@@ -0,0 +1,237 @@
+from flask import Blueprint, make_response, request, jsonify, abort
+from app import db
+from app.models.task import Task
+from datetime import datetime
+from app.models.goal import Goal
+import os, requests
+
+#VALIDATE ID
+def validate_id(class_obj,id):
+    try:
+        id = int(id)
+    except:
+        abort(make_response({"message":f"{id} is an invalid id"}, 400))
+    query_result = class_obj.query.get(id)
+    if not query_result:
+        abort(make_response({"message":f"{id} not found"}, 404))
+
+    return query_result
+
+#CREATE TASK
+task_bp = Blueprint("Task", __name__, url_prefix="/tasks")
+@task_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+    if "title" not in request_body or "description" not in request_body:
+        #TODO later wave will need 'or "completed_at" not in request_body', add above
+        return make_response({"details": "Invalid data"}, 400)
+
+    new_task = Task.from_json(request_body)
+
+    # abort(make_response)  
```

Comment:
Writing code is often messy, but in general I'm in favor of leaving commented code out. If it does need to stick around for some reason, I usually add a comment why (usually a TODO comment).

Reply:
Won't comment on every instance of this, but there's more of them.